assets/models/system/compvis-stable-diffusion-v1-4/spec.yaml

```diff
@@ -40,4 +40,4 @@ tags:
       Standard_ND96amsr_A100_v4,
       Standard_ND96asr_v4
     ]
-version: 10
+version: 11
```

Comment:
We have deci diffusion as well https://ml.azure.com/models/deci-decidiffusion-v1-0/version/6/catalog/registry/azureml

Reply:
added it